.github/workflows/Java_CI.yaml

```diff
@@ -26,6 +26,7 @@ jobs:
     - name: Build with Gradle
       run: |
         ./gradlew clean
+        ./gradlew :wrims-core:build
```

Comment:
Shouldn't this already get picked up by `./gradlew build`?

Reply:
This is related to a strange issue I recently encountered. Tracking under this issue ticket for future resolution:

https://github.com/CentralValleyModeling/wrims/issues/150

Reply:
weird!